### ISSUE TITLE
Listing all default keybindings in documentation?

### ISSUE THREAD

**Issue by kirschner** [astroidmail/astroid#323]
A minor point I realised. In the documentation https://github.com/astroidmail/astroid/wiki/Customizing-key-bindings it is written that you can generate a list of all keybindings with the `devel/get_keys.py` script. 

I was wondering if there would be an easy way to generate this list automatically for the documentation, so users do not have to run this script, but can check all default keybindings in the online documentation.

**Comment by mxmehl** [comment]
Would be nice. The problem is that the keybindings may change once in a while and that the wiki is in another git repository than astroid (so I see no way how to make this possible with a git hook).

One possibility would be to create a default keybindings file automatically with each astroid build, e.g. that scons runs `devel/get-keys.py > ~/.config/astroid/keybindings.orig`.

**Comment by gauteh** [comment]
Max writes on april 18, 2017 21:12:
> Would be nice. The problem is that the keybindings may change once in a while and that the wiki is in another git repository than astroid (so I see no way how to make this possible with a git hook).
> 
> One possibility would be to create a default keybindings file automatically with each astroid build, e.g. that scons runs `devel/get-keys.py > ~/.config/astroid/keybindings.orig`.

Yes, this is a possibility, or we might be able to set up some hook
on github that does this automatically.

**Comment by kirschner** [comment]
Hello @gauteh. Run into the issue again to find out some keybindings (had to find out what keys are still available for own shortcuts). Any chance the solution by @mxmehl or a hook can be implemented? Thank you!

**Comment by j-hui** [comment]
@kirschner there's a script under `devel/` you can run that will extract all the available commands and their default bindings from the source.

**Comment by kirschner** [comment]
@j-hui Yes, I know that. The idea was that users who do not have the devel/ files on their machines also can see the keybindings (as it is part of the documentation then). For the distribution packages that would be helpful from my perspective.

**Comment by gauteh** [comment]
That would be useful, can probably be made part of the build process
somehow.
